src/TableFooter.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const TableFooter = ({dataArray, currentPage, selectedNumber, onChange, data}) => {
+  const pageNumbersButton = []
+  for (let i = 1; i <= Math.ceil(dataArray.length/selectedNumber); i++) {
+    pageNumbersButton.push( i === currentPage ? <li className={`paginate_button number current`} key={`bottom${i}`}>{currentPage}</li> :
+      <li className={`paginate_button number`} key={`bottom${i}`}><a onClick={() => onChange(i)}>{i}</a></li>)
+  }
+  return [
+    <div key={`bottom-info`} className={`small-6 columns`}>
+      <div className={`dataTables_info`}>{dataArray.length === 0 ? `There is no search results under this query`
+        : `Found ${dataArray.length} entries filtered from ${data.length} total entries.`} </div>
```

Comment:
You could change this text to be a bit more user friendly. Something like "Showing `x` out of `y` experiments".